components/viewer/ProjectMenu.vue

```diff
@@ -78,6 +78,11 @@ const loadRemoteFile = async ({ target }: MouseEvent) => {
         });
       }
 
+      progress.value = `Loading ${target.text}...`;
+      // A hack, but otherwise the progress value never updates before loadProject is called
+      const pause = new Promise((resolve) => setTimeout(resolve, 100));
+      await pause;
```

Comment:
I did try nextTick, both sync and async, however neither could insure the text actually visually changed before loadProject took over. I suspect it's an issue of the DOM updating too quickly, but I honestly don't know.